content/kubermatic/v2.18/architecture/known_issues/_index.en.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Known Issues"
+date = 2022-07-20T12:22:15+02:00
+weight = 1
+
++++
+
+## Overview
+
+This page documents the list of known issues description and possible work arounds/solutions.
+
+### 1. OIDC user authentication issue
+
+**Problem** 
+
+OIDC user is denied to access the user cluster in the KKP with K8s version of 1.20 and below. Refer the github issue [Bug: OIDC authentication...](https://github.com/kubermatic/kubermatic/issues/9908) for detailed problem description.
+
+**Root cause**
+ 
+The KKP network policy is relying on the namespace label `kubernetes.io/metadata.name` which is automatically present on K8s 1.21 and above versions, but missing on K8s versions below 1.21. Due to the mismatch in the label selector, the access is denied.
```

Comment:
i would suggest "The KKP network policy" => "KKP API Server network policy" + link to the feature page, e.g. https://docs.kubermatic.com/kubermatic/v2.20/tutorials_howtos/networking/apiserver_policies/